docs/docs/stories.mdx

```diff
@@ -45,6 +45,19 @@ to predict the next action based on a *combination* of both the intent and
 entities (you can, however, change this behavior using the
 [use_entities](./domain.mdx#ignoring-entities-for-certain-intents) attribute).
 
+2) You do not have to deal with the specific
+intents of the messages that are extracted by the NLU pipeline. Instead, you can simply put
+the text of the user message.
+We call it end-to-end training because actual text is used for predictions.
+
+:::caution
+This feature is experimental.
```

Comment:
Are we actually raising a warning from inside the code when users are using this feature?

Reply:
hm, I guess no. I think we should do it when we know that stories are `e2e`. @joejuzl I think you added this check somewhere, could you please add the warning